src/SimplCommerce.WebHost/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -99,8 +99,12 @@ public static IApplicationBuilder UseCustomizedStaticFiles(this IApplicationBuil
 
         public static IApplicationBuilder UseCustomizedRequestLocalization(this IApplicationBuilder app)
         {
-            var cultureRepository = app.ApplicationServices.GetRequiredService<IRepositoryWithTypedId<Culture, string>>();
-            GlobalConfiguration.Cultures = cultureRepository.Query().ToList();
+            using (var scope = app.ApplicationServices.CreateScope())
```

Comment:
Why you are creating a scope here?!!

Reply:
this method is called when the application start,  no scope has been created at this stage. but DbContext is registered as Scoped. So we have to create a scope to get the DbContext